mitodl/open-discussions
# Stop sharing new code with old open

#### Pre-Flight checklist


#### What are the relevant tickets?
#3852 

#### What's this PR do?
This PR finishes removing the new TS-based packages from the old codebase. In particular, this
- adds a JS copy of SearchInput and Searchbox components to `frontends/open-discussions`

#### How should this be manually tested?
There should be no visible changes, but it is worth checking that:
1. search on `/learn` and `/learn/search` still work as expected.
